projects/table-builder/src/lib/classes/table-store.ts

```diff
@@ -57,7 +47,7 @@ export class TableStore extends ComponentStore<TableState> {
   );
 
   getMetaData$ = (key: string) : Observable<MetaData> => {
-    return this.select(this.metaData$, md => md.find(m => m.key === key ))
+    return this.select( state => state.metaData[key]  )
   }
 
   createPreSort = (metaDatas: Dictionary<MetaData>): Sort[] => {
```

Comment:
`mergeMetaDatas` methos seems to be incorrect. It should be
` mergeMetaDatas = (existingMetaData: Dictionary<MetaData>, incomingMetaData: Dictionary<MetaData>) => {
    const metaData: Dictionary<MetaData> = {};
    const incomingMetaDatas = Object.values(incomingMetaData);
    incomingMetaDatas.forEach( md => {
        const existing = metaData[md.key] ?? existingMetaData[md.key];
        if(!existing) {
          metaData[md.key] = { ...md, noExport: md.customCell }
        } else {
          metaData[md.key] = this.mergeMeta(existing,md);
        }
    });
    return metaData;
  }`

Reply:
I think this should fix it, although I suspect there may be a bug as now it doesn't auto add the custom cell field